content/docs/console/integrations/pulumi-button.md

```diff
@@ -23,7 +25,8 @@ There are two steps to create a button:
 
 The Pulumi button works with template projects hosted in public GitHub repositories or Gists. A template is a Pulumi project that has the required `Pulumi.yaml` file, which describes the project. The template project can be in the root of the GitHub repository or within a subdirectory. Multiple projects can be hosted within subdirectories of a single repository.
 
-The `Pulumi.yaml` file can optionally contain a `template` section, which typically includes a `config` section used to specify any required config values for the project. Each config value can have a `description` and `default` value. Config values can also have a `secret` property, which can be set to `true` to indicate the config value is a [secret]({{< relref "../config.md#secrets" >}}).
+The `Pulumi.yaml` file can optionally contain a `template` section, which typically includes a `config` section used to specify any required config values for the project. Each config value can have a `description` and `default` value. Config values can also have a `secret` property, which can be set to `true` to indicate the config value is a
+[secret]({{< ref "/docs/reference/config.md#secrets" >}}).
```

Comment:
Why the change from `relref` to `ref`? (As an aside, I think we should do a pass throughout the site and change any `ref`s to just be `relref`, unless there is a need for the URL to be a fully qualified URL.

Reply:
Let's sync up about this, since if we kept them as `relref` it seems difficult to maintain if it becomes `"../../reference/config.md"`. Just having a qualified `ref` seems cleaner IMHO, though when we do a pass over all of our Service doc content, let's revisit this conversation.